redash/models/__init__.py

```diff
@@ -64,6 +65,58 @@ def get(self, query_id):
 scheduled_queries_executions = ScheduledQueriesExecutions()
 
 
+@python_2_unicode_compatible
+class TableMetadata(TimestampMixin, db.Model):
+    id = Column(db.Integer, primary_key=True)
+    data_source_id = Column(db.Integer, db.ForeignKey("data_sources.id"))
+    table_exists = Column(db.Boolean, default=True)
+    table_name = Column(db.String(255))
+    table_description = Column(db.String(4096), nullable=True)
```

Comment:
This is a table object, so I guess the `table_` prefix for columns is a bit redundant? (same goes for `column_` in the Column model).